Sources/protoc-gen-swift/EnumGenerator.swift

```diff
@@ -22,6 +22,8 @@ private let unrecognizedCaseName = "UNRECOGNIZED"
 
 /// Generates a Swift enum from a protobuf enum descriptor.
 class EnumGenerator {
+  private static let requiredProtocolConformancesForEnums = ["Swift.CaseIterable"].joined(separator: ", ")
```

Comment:
Should we put a todo here to move this onto enum when we can do breaking changes?